pkg/controllers/provisioningv2/rke2/rkecluster/controller.go

```diff
@@ -59,16 +55,20 @@ func (h *handler) UpdateSpec(_ string, cluster *v1.RKECluster) (*v1.RKECluster,
 }
 
 func (h *handler) OnChange(obj *v1.RKECluster, status v1.RKEClusterStatus) (v1.RKEClusterStatus, error) {
+	condition := rke2.Ready
```

Comment:
nit: this variable name is vague and potentially confusing without an IDE

Reply:
Will update.